### ISSUE TITLE
Update badge colors

### ISSUE THREAD

**Issue by GarthDB** [adobe/spectrum-css#1350]
A couple badge colors do not meet accessibility requirements.

synced to jira: https://jira.corp.adobe.com/browse/SDS-11272

**Comment by pfulton** [comment]
Should be fixed, as this component has been migrated to use `@adobe/spectrum-tokens`
